src/utils/fileExports.js

```diff
@@ -16,6 +18,18 @@ const { ActivityFilterTypes } = core.activity;
 
 const formatDate = (dateTime) => dateTime.toFormat(`dd.LL.yyyy`).toString();
 
+const loadPaymentNote = (txHash) => {
+  return from(resolveTxHash(txHash).then((note) => note || '-'));
+};
+
+const getPaymentNotes = (transactions) => {
+  const paymentNotes$ = from(transactions).pipe(
+    mergeMap((tx) => loadPaymentNote(tx.txHash), 1),
```

Comment:
it would be worth to try with different values like 2,5 or 10
`mergeMap((tx) => loadPaymentNote(tx.txHash), 2)` as an example, it should increase perforamance and potentionally still not crash

Reply:
That's a nice idea.

Reply:
I will try this :)